script/Arcueid/Day 4/04_00_ARC04_10.txt

```diff
@@ -762,12 +762,12 @@ She doesn't need to pull me back this time.
 [sha:70702a28e88710ede3aa814743b40ae090481b65]{
 -- Page 852, Offset 7926.
 -- アルクェイドは路地裏の時と同じように、右手を振るって炎を弾き流す。
-Arcueid swings her right hand and blows away the flames, just as she did in the alleyway.
+Arcueid swings her right hand, just as she did in the alleyway, and scatters the flames.
```

Comment:
scatters > disperses 
maybe it's just me but "scatters" sounds like she's spreading it rather than blocking it.